core/src/main/java/org/testcontainers/utility/MountableFile.java

```diff
@@ -241,8 +243,23 @@ private String extractClassPathResourceToTempLocation(final String hostPath) {
         return tmpLocation.getAbsolutePath();
     }
 
+    private boolean isOnBitbucket() {
+        final String bbCloneDir = System.getenv(BITBUCKET_CLONE_DIR_ENV);
```

Comment:
call getBitbucketCloneDir() here?

Reply:
probably some other vars will be required so i'd prefer not to join BB checking and getting some BB variable